custom-shipping-methods-for-woocommerce.php

```diff
@@ -121,6 +121,7 @@ public function is_plugin_active( $plugin ) {
 		 * @version 1.1.0
 		 * @since   1.0.0
 		 */
+		protected $core;

```

Comment:
This line must appear above the comment.

The comment was for function.

Also, please add comment for the variable